echopype/convert/set_groups_azfp.py

```diff
@@ -512,18 +500,25 @@ def set_vendor(self) -> xr.Dataset:
         unpacked_data = self.parser_obj.unpacked_data
         parameters = self.parser_obj.parameters
         ping_time = self.parser_obj.ping_time
+        PHASE_PARAMS = ["burst_interval", "pings_per_burst", "average_burst_pings"]
+        PHASE_FREQ_PARAMS = [
+            "dig_rate",
+            "range_samples",
+            "range_averaging_samples",
+            "lock_out_index",
+            "gain",
+            "storage_format",
+        ]
```

Comment:
Since the scope of these two variables is no longer global to the module, I suggest using lower-case names instead.

Reply:
Updated, changed the names to lower-case.